docs/developing/troubleshooting.md

```diff
@@ -35,16 +40,20 @@ very likely you are seeing this error.
 If possible, configure the app to use a base URL of `''`, literally
 the empty string. Then it will send HTTP redirects without
 specifying a base URL. If that isn't possible, Sandstorm apps should
-look at the
+look at the `Host:` header for a base URL.
```

Comment:
https://docs.sandstorm.io/en/latest/developing/troubleshooting/ is now up
to date. Thanks again!